lib/toSyllabicSymbols.js

```diff
@@ -0,0 +1,147 @@
+const toUnicode = require( './toUnicode' )
+const { getRegexGroup } = require( './regex-utils' )
+
+// These have no impact on weight, therfore remove before processing
+const zeroWeightSigns = [
+  '੍ਰ',
+  'ੵ',
+  'ੑ',
+  '੍ਵ',
+  '੍ਹ',
+  '੍ਟ',
+  '੍ਨ',
+  '੍ਯ',
+  '੍ਚ',
+  '੍ਤ',
+  '੍',
+  'ਿ',
+  'ੁ',
+  'ਂ',
+  'ਃ',
+  '।',
+  '॥',
+  '☬',
+  '਼',
+  '❁',
+]
+
+// For the rest, need to analyze the string
+// Base characters counts as one syllable (light)
+// Base characters with any number of long (deeragh) sounds counts as two syllables (heavy)
+const syllableSymbol = {
```

Comment:
```suggestion
const syllableSymbols = {
```
It is a map of many symbols